core/pom.xml

```diff
@@ -34,6 +34,10 @@
       <artifactId>galley-api</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.commonjava.maven.galley</groupId>
```

Comment:
What's this new dependency used for? If possible, I'd like to reduce its scope to `test` or maybe eliminate it from the core module build, since it shouldn't really be necessary for a user to have a dependency on the httpclient transport in order to use the core functions...in principle, they should be able to use galley with a ssh, ftp, zip, or even just file transports, without http. I know it's unlikely, but still...